masim.c

```diff
@@ -130,7 +134,7 @@ static void do_seq_ro(struct access *access, int batch)
 {
 	struct mregion *region = access->mregion;
 	char *rr = region->region;
-	size_t offset = access->last_offset;
+	uint64_t offset = access->last_offset;
```

Comment:
`size_t` is 8 bytes on 64-bit machines, right?  Would we want to run 64-bit access pattern on 32-bit machine?  If not, why we need this change?